### ISSUE TITLE
Upgrading to 3.0

### ISSUE THREAD

**Issue by tdtadeu** [mhenrixon/active_campaign#35]
I opened this issue so we can debate it here. I accidentally started the discussion at #33, but I didn't want to go on there since that issue asks for something different, yet related.

V3 is RESTful, at last! https://developers.activecampaign.com/reference

@mhenrixon Should we maintain the same structure of method call using ``method_missing`` in the main module, or could I suggest we create resource classes to map their new API?

**Comment by mhenrixon** [comment]
@tdtadeu I'm fine with using something more _real_ for v3. I wanted to do it for the longest time. That said I wouldn't go too crazy with it.

**Comment by mikevalen** [comment]
Are there any plans on updating this gem to use V3 of the API?

Thanks!

**Comment by mhenrixon** [comment]
Yeah but there is no time unfortunately. If I can manage to free up enough time I'll do it but right now I have to focus on paid work since I don't use Active Campaign at work anymore.

**Comment by kjanoudi** [comment]
Possible to issue a gem release?

**Comment by mhenrixon** [comment]
@kjanoudi needs a few more endpoints first

**Comment by arthurnobrega** [comment]
@mhenrixon and @kjanoudi I can help you guys finishing this integration. Can you list me what is missing?
